openshift_scalability/config/ocp-logtest.yaml

```diff
@@ -0,0 +1,19 @@
+projects:
+  - num: 1
+    basename: logtest
+    ifexists: delete
+    tuning: default
+    templates:
+      - num: 1
+        file: ./content/logtest/logtest-rc.json
+        parameters:
+         - REPLICAS: "3"
```

Comment:
They are exposed in a configmap - see logtest-rc.json.   They can be updated there pre-deploy or updated and the rc scaled down/up.   Would you like to see them in the cluster-loader configmap as well?